live-examples/js-examples/string/string-iterator.html

```diff
@@ -0,0 +1,13 @@
+<pre>
+<code id="static-js">const str = "The quick red fox jumped over the lazy dog's back.";
```

Comment:
Use single quotes for strings, please: https://github.com/mdn/interactive-examples/blob/master/JS-Example-Style-Guide.md#single-quotes

Reply:
Done